test/crs/domains.jl

```diff
@@ -37,125 +37,75 @@
     end
 
     # backward
-    atol = if C <: Lambert
-      T === Float32 ? 1.0f-2° : 1e-4°
-    elseif C <: Behrmann
-      T === Float32 ? 1.0f-2° : 1e-4°
-    elseif C <: GallPeters
-      T === Float32 ? 1.0f-1° : 1e-4°
-    elseif C <: Robinson
-      T(1e-3) * °
-    elseif C <: Albers
-      T === Float32 ? 1.0f-1° : 1e-5°
-    elseif C <: EqualEarth && T === Float64
-      1e-5°
-    else
-      nothing
+    if C <: TransverseMercator
+      # TODO: fix backward implementation
+      continue
+    elseif C <: Robinson && T == Float64
+      # https://github.com/JuliaEarth/CoordRefSystems.jl/issues/55
+      continue
     end
 
-    if C <: OrthoNorth
-      # coordinates at the singularity of the projection (lat ≈ 90) cannot be inverted
-      for lat in T.(1:89), lon in T.(-180:180)
-        c1 = LatLon(lat, lon)
-        if indomain(OrthoNorth, c1)
-          c2 = convert(OrthoNorth, c1)
-          c3 = convert(LatLon, c2)
-          @test allapprox(c3, c1)
-        end
+    for lat in T.(-90:90), lon in T.(-180:180)
+      if C <: EqualEarth && lat == -90
+        # https://github.com/JuliaEarth/CoordRefSystems.jl/issues/243
+        continue
+      elseif C <: Orthographic && lat == 0
+        continue
```

Comment:
It would be nice to have a comment for this branch too.

Reply:
Sorry, I for some reason thought this case didn't need an explanation but clearly I was wrong. Fixed now.